[1.15.0] rest-framework-jwt breaks if blacklist app is not installed

New version forces users to have blacklist app installed even if the feature is not being used.

Eg. of failed test:

> psycopg2.errors.UndefinedTable: relation "rest_framework_jwt_blacklistedtoken" does not exist
E               LINE 1: SELECT (1) AS "a" FROM "rest_framework_jwt_blacklistedtoken"...


I assume the issue is [here](https://github.com/Styria-Digital/django-rest-framework-jwt/blob/master/src/rest_framework_jwt/authentication.py#L70).

It wold be desirable to add a check if the app is installed and only run it if it is.

I can provide a fix if we agree on the solution :smile:

--- a/changelog.d/35.bugfix.md
+++ b/changelog.d/35.bugfix.md
@@ -1,0 +1,1 @@
+Added check in authentication if blacklist app is installed before checking if any Blacklisted tokens exist

--- a/src/rest_framework_jwt/authentication.py
+++ b/src/rest_framework_jwt/authentication.py
@@ -4,6 +4,7 @@ from __future__ import unicode_literals
 
 import jwt
 
+from django.apps import apps
 from django.contrib.auth import get_user_model
 from django.utils.encoding import force_str
 
@@ -67,9 +68,10 @@ class JSONWebTokenAuthentication(BaseAuthentication):
         except MissingToken:
             return None
 
-        if BlacklistedToken.objects.filter(token=force_str(token)).exists():
-            msg = _('Token is blacklisted.')
-            raise exceptions.PermissionDenied(msg)
+        if apps.is_installed('rest_framework_jwt.blacklist'):
+            if BlacklistedToken.objects.filter(token=force_str(token)).exists():
+                msg = _('Token is blacklisted.')
+                raise exceptions.PermissionDenied(msg)
 
         try:
             payload = self.jwt_decode_token(token)

